microsoft-edge/devtools-guide-chromium/whats-new/2022/03/devtools.md

```diff
@@ -0,0 +1,177 @@
+<!-- Subtitle: Get the Edge Developer Tools extension for VS today! -->
+
+Following up on the success of the Visual Studio Code integration, you can now also embed the Microsoft Edge Developer Tools in Microsoft Visual Studio to debug your ASP.NET projects live.  Download the [Microsoft Edge Developer Tools for Visual Studio](https://aka.ms/edgetools-for-vs) and try it out. 
+
+To try this feature:
+1. Make sure Visual Studio 2022 and the ASP.NET workload are installed.
+1. Set up Web Live Preview as the default Web Forms Designer, as described at [Microsoft Edge Developer Tools for Visual Studio](https://aka.ms/edgetools-for-vs).
+1. In an ASP.NET project, open a webpage of the project in the **Design** window.
+1. In the upper left of the **Design** window, click the **Open Edge DevTools** (![Open Edge DevTools icon.](../../media/2022/03/open-edge-dev-tools-v-s-icon.png)) button:
+
+![An ASP.NET project, opening Edge DevTools.](../../media/2022/03/devtools-extension-v-s-web-forms-designer.png)
+
+Edge DevTools for Visual Studio opens, with the **Elements** tool selected:
+
+![Microsoft Edge Developer Tools for Visual Studio: Elements tool of DevTools.](../../media/2022/03/devtools-extension-visual-studio-elements.png)
+
+The **Network** tool is also available by default:
+
+![Microsoft Edge Developer Tools for Visual Studio: Network tool of DevTools.](../../media/2022/03/devtools-extension-visual-studio-network.png)
+
+The **Inspect tool** (![Inspect tool icon.](../../media/2022/03/v-s-edge-devtools-inspect-tool-icon.png)) and **Toggle screencast** (![Toggle screencast icon.](../../media/2022/03/v-s-edge-devtools-toggle-screencast-icon.png)) are available, and the **More Tools** (![More Tools icon.](../../media/2022/03/more-tools-v-s-icon.png)) menu offers the [Issues](../../../issues/index.md), [Network conditions](../../../network-conditions/network-conditions-tool.md), and [Network request blocking](../../../network-request-blocking/network-request-blocking-tool.md) tools.
+
+See also:
+
+* [Edge Developer Tools for Visual Studio (Preview)](https://devblogs.microsoft.com/visualstudio/?p=237066&preview=1&_ppp=7aa7aef54f), at Visual Studio DevBlogs.
+* [Edge DevTools extension for Visual Studio](../../../../visual-studio/index.md#edge-devtools-extension-for-visual-studio).
+
+
+<!-- ====================================================================== -->
+## Layers in 3D View
+
+<!-- Title: Layers in 3D View -->
+<!-- Subtitle: The Layers tool isn't going away - find it in the 3D View tool today. -->
+
+Since [Microsoft Edge 88](../../2020/11/devtools.md#composited-layers-are-now-in-3d-view), the **3D View** tool has provided an improved version of the **Layers** tool.  Now, in Microsoft Edge 99, the standalone **Layers** tool has been removed, and contains a link that redirects you to the **Composited Layers** view within the **3D View** tool.  You can find all the functionality from the **Layers** tool in the **3D View** tool.
```

Comment:
Realizing that "Layers tool has been removed and contains a link..." doesn't really make sense. Let's change it to "Now, in Microsoft Edge 99, the functionality in the standalone **Layers** tool has been removed and instead contains a link redirecting you to the **Composited Layers** view within the **3D View tool. You can find all the features previously in the **Layers** tool and more in the **3D View** tool.